assets/components/atoms/link/link.scss

```diff
@@ -1,27 +1,31 @@
 @charset 'utf-8';
 
-a:not([class*="btn"]) {
+a,
+a:hover,
+a:active { text-decoration: none; }
+
+a:not([class*="btn"]):not([class*="teaser"]):not([class*="card"]):not(.social-icon) {
```

Comment:
this kind of breaks breadcrumbs. It needs some refactoring to use the new pretty link visual

Reply:
Breadcrumb fixed with the footer branch (merged) 😉